subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/suite/filter/AnnotationsFilter.kt

```diff
@@ -0,0 +1,29 @@
+package com.avito.instrumentation.suite.filter
```

Comment:
A little overview. Now we can exclude and include tests by annotations and signatures.